avro-fastserde/src/main/java/com/linkedin/avro/fastserde/Utils.java

```diff
@@ -33,6 +33,14 @@ public static boolean isAvro14() {
     return AvroCompatibilityHelper.getRuntimeAvroVersion().equals(AvroVersion.AVRO_1_4);
   }
 
+  public static boolean isAbleToSupportAlternativeStrings() {
+    return AvroCompatibilityHelper.getRuntimeAvroVersion().laterThan(AvroVersion.AVRO_1_6);
+  }
+
+  public static boolean isAbleToSupportJavaStrings() {
```

Comment:
I believe Avro 1.6 added support for Java strings (as opposed to Utf8) in 1.6, and that this applied to all strings (fields, map keys...), then, in 1.7, they further extended this support to other types (URL, URI, etc.). That's why there's a method that checks for > 1.5 and another for >1.6.

I can rename them but I'm not sure what would be a better name. "Stringable prop" does not sound very intuitive either, to me...

Reply:
I think there  is some disconnect here.
There are  two types of String related support in Avro.
`avro.java.string` was introduced earlier and as you mentioned that it can applied to  all the string fields to specify the right Java type during deserialization, and it is supported in both GenericDatumReader and SpecificDatumReader.
Another string type related feature is called `Stringable Prop`, which is more advanced, since you could not just specify more Java types than `String` for `string` fields, and it is only applicable for SpecificDatumReader, which means these props will be ignored while using GenericDatumReader, and inside Vanilla Avro, it is being called `Stringable Prop`.
That is the original reason I want to clarify which function is for which feature.

Reply:
Ok I hadn't realized this was how it was called in vanilla Avro. Then it SGTM to rename `isAbleToSupportAlternativeStrings` to `isAbleToSupportStringableProp` as you suggested.